src/search/node/FindInFilesDomain.js

```diff
@@ -324,9 +325,13 @@ maxerr: 50, node: true */
             setTimeout(fileCrawler, 1000);
             return;
         }
-        var i = 0;
+        var i = 0,
+            contents = "";
         for (i = 0; i < 10 && currentCrawlIndex < files.length; i++) {
-            getFileContentsForFile(files[currentCrawlIndex]);
+            contents = getFileContentsForFile(files[currentCrawlIndex]);
+            if (contents) {
+                cacheSize += contents.length;
```

Comment:
@prafulVaishnav @nethip Fwiw I think this number is going to be inaccurate -- if I'm reading this code correctly, every time you re-index a file that was updated, you're going to add it to the existing cacheSize total even though it's overwriting its old entry.  So if a file is updated often, it could get counted in the cacheSize 100 times instead of just once.